### PR TITLE
Log version with routes

### PR DESCRIPTION
**Pull request by olamilekan000**
Made some corrections as to how routes are being logged

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -32,18 +32,46 @@ exports.getRoutes = routerStack => {
...
       const { handle } = stacks;
-      const version = stacks
+
+      const strigifiedRegex = stacks
         .regexp
-        .toString()
-        .replace(/[fast_car:false\\?i=|$)()/^true]/ig, '');
+        .toString();      
+
+      const baseurl = strigifiedRegex.search(/api/);
```
Comment (BolajiOlajide) [inline src/utils.js]: this has the assumption that the base url will always start with `api` but it could be anything

Comment (olamilekan000) [reply src/utils.js]: Anything like, cos I feel most Base URL always starts with api. I'd like to know what other names the base url could be called.

Comment (olamilekan000) [reply src/utils.js]: And then agian, did you delete your twitter account?

Comment (BolajiOlajide) [reply src/utils.js]: Yeah, so we have to programmatically get it out. We can't always assume it's `api`

I deactivated my twitter account temporarily.

Comment (olamilekan000) [reply src/utils.js]: Alright. I'll look into it.

Comment (olamilekan000) [reply src/utils.js]: I just made a change to the PR

**Pull request merged**